tests/network/test_services.py

```diff
@@ -36,41 +36,3 @@ def test_set_service_timeout():
     assert updated == 3
 
     set_service_timeout(original)
```

Comment:
Might be nice to add some tests for this ;).

Reply:
Added tests in #25